packages/db-manager/__tests__/index.js

```diff
@@ -3,7 +3,7 @@ jest.mock('knex')
 
 const knex = require('knex')
 
-const middy = require('@middy/core')
+const middy = require('../../core')
```

Comment:
any reason for changing this?

Reply:
Just to make it consistent with all the other modules. This ensures they're all being tested with the same version of core as the rest.